### ISSUE TITLE
tyk-sync improvements

### ISSUE THREAD

**Issue by rewsmith** [TykTechnologies/tyk-docs-hugo#1371]
## Relevant Document

https://github.com/TykTechnologies/tyk-docs/blob/master/tyk-docs/content/advanced-configuration/manage-multiple-environments/tyk-sync.md

## Describe the error

The current tyk-sync docs state that it is only available as go, when in fact it can also be run as docker.  So need to include instructions on how to install and run as docker.  The examples will also need to be updated to include docker specific examples, including how to access files on the host machine using something like docker bind mounts.

**Comment by sedkis** [comment]
Here's an example using a Dockerized Tyk-Sync, maybe we can use this as a basis for any documentation?:
https://github.com/sedkis/tyk/blob/master/scripts/tyk-sync-push-pull.bash

**Comment by rewsmith** [comment]
Yeah, I think we need to provide examples which we can base on your example
